### DISCUSSION TITLE
Parejas TG1

### DISCUSSION THREAD

**Discussion by IchottMano** [Exploratorio-DCC-PUC/Syllabus#523] [General]
Usen esta discusión para buscar parejas para la Tarea Grande 1

**Comment by Rlopeze** [comment]
Hola busco pareja para la Tarea!

**Reply by tomytable** [reply]
Hola, conseguiste pareja? a mi igual me falta

**Reply by Rlopeze** [reply]
naa, buenaa metee, escribeme al +56994351610
